src/main/java/Duke/Tasks/Task.java

```diff
@@ -0,0 +1,82 @@
+package Duke.Tasks;
+
+import Duke.Tool.Storage;
+import Duke.Tool.TaskList;
+import Duke.Ui;
+
+
+/**
+ * Represents the Task class
```

Comment:
It would be good to provide more descriptive details in this JavaDocs line, so that reviewers can understand this class better. Currently, I think this is rather vague.